### PR TITLE
Scenario column

### PR DESCRIPTION
**Pull request by ningning621**
- Added labels to list display so that it is no longer just "cryptic numbers"
- Add and save button full integrated, checked that generated file matched list display
- Both buttons validates that json object fields are correct (ie 1 bot/fields aren't empty)

### CODE REVIEW

File: gui/static/js/components/scenarios.js [hunk 1]
```diff
@@ -42,35 +44,39 @@ export default class ScenariosItem extends React.Component {
...
-        // .catch(function (error) {
-        //     console.log(error);
-        // });
+        console.log("numBots:" + this.state.numBots);
+        if (this.state.numBots != 1) alert("There must be exactly one bot!");
+        else {
+            var filename = this.state.filename;
+
+            //Implemented the same way as Python script saving. This might not work so use axios code if it doesn't
+            //event . preventDefault ()
```
Comment (chynu) [inline gui/static/js/components/scenarios.js]: Please remove if it is not necessary.

**Review by chynu — CHANGES_REQUESTED** [review]
Hm, for some reason the GUI still shows the "cryptic numbers"?

File: gui/static/js/components/scenarios.js (hunk 1 above)
Comment (yangdanny97) [reply gui/static/js/components/scenarios.js]: We don't know if it's necessary or not until we test it, and it's not ready to test just yet.

**Comment by yangdanny97** [comment]
Cryptic numbers = values for various fields in an example scenario object json (a clean interface for this has not been made yet, but it'll make sense once it's made). Please ignore for now, it doesn't really affect anything.

File: gui/static/js/components/scenarios.js (hunk 1 above)
Comment (ningning621) [reply gui/static/js/components/scenarios.js]: deleted; I realized that I assigned filename a value in the constructor

**Comment by chynu** [comment]
Oh I think I must have forgotten to do npm run webpack at some point and it didn't load. It works now !!

Anyway, there seems to have been some confusion wrt static/dynamic loading and `index.html` and `index2.html`. 

Please remove `index2.html` and always test by doing `npm run webpack` and `PYTHONPATH=.. python3 main.py` and going to `localhost:8080/gui` on the browser, _not_ by loading the static HTML file itself.

Otherwise, LGTM! :)

**Review by chynu — APPROVED** [review]
:+1:

**Pull request merged**